lib/places.js

```diff
@@ -18,9 +20,7 @@ function tagFromVersion (version) {
 export function localPlace (opts) {
   const p = placesJson.localPlace;
   const { version } = opts;
-  const tag = tagFromVersion(version);
-  Object.assign(opts, { tag });
-  const atHome = path.join(cachePath, p);
+  const atHome = IGNORE_TAG ? path.join(cachePath, p) : path.join(cachePath, tagFromVersion(version), p);
```

Comment:
Is there a good way to verify this won't break existing consumers? 

In a similar thread to other PR, this env var would be great to them document in the README.

Reply:
Hi @leerob, based on my tests this wasn't breaking anything what I don't understand is why I removed the `Object.assign(opts, { tag })`. BTW the only way to be 100% sure would be creating some integration tests

Reply:
BTW In created https://github.com/vercel/pkg/pull/1053 to document this

Reply:
Merged! And agreed, removing Object assign seemed off.  Maybe when `expand`ing the object below it doesn't matter?

Reply:
Yep by checking it again I can confirm that I removed it because I have removed the tag from the template so it's useless to pass it to the expand function.